src/org/scada_lts/dao/MultiChangesHistory.java

```diff
@@ -0,0 +1,15 @@
+package org.scada_lts.dao;
+
+import org.scada_lts.dao.model.multichangehistory.MultiChangeHistoryValues;
+import org.scada_lts.service.model.MultiChangeHistoryDTO;
+
+import java.util.List;
+
+/**
+ * @author grzegorz.bylica@abilit.eu on 15.10.2019
+ */
+public interface MultiChangesHistory {
+
+    List<MultiChangeHistoryValues> getHistory(String viewAndCmpId);
+    void prcAddCmpHistory(Integer userId, String xIdViewAndIdCmp, String interpretedState, Long scadaTime, String values);
```

Comment:
Is it possible to change name of method to more readable -> prcAddCmpHistory?

Reply:
improved